best-practices/README.md

```diff
@@ -217,6 +217,7 @@ JavaScript
 HTML
 ----
 
+* Use the [W3C's Markup Validation Service][html-validator] to validate HTML
```

Comment:
Are we expected to run this manually on every page of a project?

Reply:
I believe, and @tysongach correct me if I am wrong, it would make the most sense to run it on whatever HTML is being run by the browser. Ideally, I would want to deploy to, say a Heroku Review Branch, and then send that URL over to the W3C's Markup Service.